src/rules/noAnyRule.ts

```diff
@@ -42,7 +43,10 @@ export class Rule extends Lint.Rules.AbstractRule {
 
 class NoAnyWalker extends Lint.RuleWalker {
     public visitAnyKeyword(node: ts.Node) {
-        this.addFailureAtNode(node, Rule.FAILURE_STRING);
+        const fix = this.createFix(
+            this.createReplacement(node.getStart(), node.getWidth(), "{}"),
```

Comment:
It cannot *really* be auto-fixed, but I see a lot of people using `any` when they mean `{}`, i.e. a type they don't care about and don't want to access properties on that is a super type of everything. I agree, it's not all that great, but maybe better than no fix?